usagov/usagov-2021
# USAGOV-382-Mobile-Navigation

## Deployment
You may need to install & enable the simplify_menu module
https://www.drupal.org/project/simplify_menu
## How to Test
First confirm that you can open and close the mobile menu and use the links to navigate around. make sure that it does not cause any unwanted changes in the header on mobile and on desktop.

The mobile nav renders certain items depending on how deep into the site structure the current page is. To make sure it is displaying the proper items, start by finding a page that goes deep into the site structure. You can find one easily by going to  Structure>Menus>Left Menu English and look for a deep menu item.

Now pull up this image:
https://cm-jira.usa.gov/secure/attachment/320086/Reimagined_Mobile_Navigation_1A.png
On the homepage, the mobile nav should look like Panel 1.
On a top-level nav page, it should look like Panel 2.
On a nav page 2 levels deep, it should look like Panel 3.
On a nav page 3 levels deep, it should look like Panel 4.
On a nav page 4 levels deep, it should look like Panel 5.
On a nav page 5 levels deep, it should look like Panel 6 (Confirm that it does not show the nav page from 2 levels deep)

Next visit the Life Events section and confirm that the mobile looks like this image:
https://cm-jira.usa.gov/secure/attachment/320085/Reimagined_Mobile_Navigation_1B.png



## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-382

## Description

Using simplify_menu module because it reduces complexity of template logic


## Checklist for the Developer



## Checklist for the Peer Reviewers
